<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpReceiver</class>
 <widget class="QWidget" name="UdpReceiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UdpReceiver</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Тип соединения</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_AutoConnection">
        <property name="text">
         <string>AutoConnection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_DirectConnection">
        <property name="text">
         <string>DirectConnection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_QueuedConnection">
        <property name="text">
         <string>QueuedConnection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_BlockingQueuedConnection">
        <property name="text">
         <string>BlockingQueuedConnection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>waitForReadyRead</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
